src/instance.ts

```diff
@@ -1501,6 +1501,7 @@ promisifyAll(Instance, {
     'getBackupsStream',
     'getTablesStream',
     'getAppProfilesStream',
+    'view',
```

Comment:
This needs to be added because calling `view` should return an Authorized View. It should not return a promis that resolves to an authorized view.